website/mails/mails.py

```diff
@@ -275,6 +275,12 @@ def get_english_article(word):
     'digest', subject='OSF Notifications',
     categories=['notifications', 'notifications-digest']
 )
+
+DIGEST_REVIEWS_MODERATORS = Mail(
+    'digest_reviews_moderators',
+    subject='Today\'s submissions to ${provider_name}',
```

Comment:
Did you confirm when these are sent on prod? The beat timing is set differently there. Mostly concerned that "Today's" might be off... Like if the email is sent at noon it would be today and yesterday till noon's changes?

maybe Last 24 hours submissions or something. Has product weighed in on this specifically?